app/src/main/java/com/github/houseorganizer/houseorganizer/shop/FirestoreShopList.java

```diff
@@ -112,12 +101,26 @@ private static List<Map<String, Object>> convertItemsListToFirebase(List<ShopIte
             itemMap.put("pickedUp", item.isPickedUp());
             items.add(itemMap);
         }
+
         return items;
     }
 
-    private static List<ShopItem> convertFirebaseListToItems(List<Map<String, Object>> list){
+    public Task<DocumentSnapshot> refreshItems() {
+        if(onlineReference == null)
+            return Tasks.forCanceled();
+
+        return onlineReference.get().continueWith( r -> {
+            if(!r.isSuccessful())
+                return null;
+            DocumentSnapshot snap = r.getResult();
+            setItems(convertFirebaseListToItems((List<Map<String, Object>>) snap.get("items")));
+            return snap;
+        });
+    }
```

Comment:
from what i found this is only used in testing, does it make sense to have it ?

Reply:
I don't know. I copy pasted the fix from Sandra in order to finally get my tests passing